rllib/env/utils.py

```diff
@@ -75,6 +94,12 @@ def _gym_env_creator(env_context: EnvContext, env_descriptor: str) -> gym.Env:
         env_descriptor: The env descriptor, e.g. CartPole-v1,
             ALE/MsPacman-v5, VizdoomBasic-v0, or
```

Comment:
Can we update the docstring about env_descriptor to say that it can be type or str and how they differ?

Reply:
done